src/eager/python/ortmodule/torch_inline_extensions/.ninja_log

```diff
@@ -0,0 +1,61 @@
+# ninja log v5
```

Comment:
Do we need to check-in the ninja files? Did you accidentally include them?

Reply:
no it is accidently got included...